README.md

```diff
@@ -1,216 +1,265 @@
 [![official project](http://jb.gg/badges/official.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# [Compose Multiplatform](https://github.com/JetBrains/compose-multiplatform) Mobile Application
+# Compose Multiplatform mobile application
 
```

Comment:
Andrey suggested to have a block that helps navigate to all Get started templates and main repo so that all repos are connected and users could easily navigate between them. I think that this is a good idea and suggest to implement.

Reply:
AFAIU, we'll only have this and desktop tutorial updated. Do you think it's necessary to give links to the rest of the stuff?

Reply:
I've added a What's next section, similar to the one that Compose Desktop has. Hope it solves the problem

Reply:
I think that it's important to mention at the top at least the desktop case. I've added a suggestion for the intro. Please take a look.